content/en/docs/reference/node/kubelet-files.md

```diff
@@ -0,0 +1,114 @@
+---
+content_type: "reference"
+title: Files written and read by the kubelet
+weight: 42
+---
+
+The {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} is mostly a stateless
+process running on a Kubernetes {{< glossary_tooltip text="node" term_id="node" >}}.
+This document outlines files that kubelet reads and writes.
+
+Kubelet uses the {{< glossary_tooltip text="control plane" term_id="control-plane" >}} as
+the source of truth on what needs to run on the Node, and the
+{{<glossary_tooltip text="container runtime" term_id="container-runtime">}} to retrieve
+the current state of containers. Kubelet also relies on reading
+cgroups and various system files to collect metrics. However, there are still a few other files
+that are used by the kubelet.
+
+## Files
+
+### Configuration
+
+#### Kubelet configuration files
+
+The path to the kubelet configuration file can be configured
+using the command line argument `--config`. kubelet also supports
+[drop-in configuration files](/docs/tasks/administer-cluster/kubelet-config-file/#kubelet-conf-d)
+to enhance configuration.
+
+#### Certificates
+
+Certificates are typically located at `/var/lib/kubelet/pki`,
+but can be configured using the `--cert-dir` kubelet command line argument.
+Names of certificate files are also configurable.
+
+#### Systemd unit settings
+
+When kubelet is running as a systemd unit, some kubelet configuration may be declared
+in systemd unit settings file. Typically it includes:
+
+- command line arguments to [run kubelet](/docs/reference/command-line-tools-reference/kubelet/)
+- environment variables, used by kubelet or [configuring golang runtime](https://pkg.go.dev/runtime#hdr-Environment_Variables)
+
+### State
+
+#### Manifests
+
+Manifests for static pods are typically located in `/etc/kubernetes/manifests`.
+Location can be configured using the `staticPodPath` kubelet configuration option.
+
+#### Checkpoint files
+
+All resource managers keep the mapping of Pods to allocated resources in state files.
+State files located in the kubelet's root directory,
+typically `/var/lib/kubelet`. Kubelet root directory can be configured
```

Comment:
Here's how I'd write it:
```suggestion
State files are located in the kubelet's base directory, also termed the _root directory_
(but not the same as `/`, the node root directory). You can configure the base directory
for the kubelet
```

(aside: I wonder if we can deprecate `--root-directory` in favor of a `kubeletBasePath` configuration option, as it is hard to disambiguate the two meanings of _root_)